profile.php

```diff
@@ -10,8 +10,10 @@
 <body>
 	<?php
 	echo "Welcome\n";
-	echo $_SESSION['name'];
-	echo $_SESSION['class'];
+	echo $_SESSION['name']." ";
```

Comment:
Write all this in HTML so that you can style it later

Reply:
👍